app/AvailablePlugin/EmailWidget/Config/Schema/schema.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" ?>
+<!--
+  COmanage Registry Self Service Email Widget Plugin Database Schema
```

Comment:
No worries - I like nitpicky. I'll get these modified.

Reply:
Done. Note that this change is reflected all the way up and down the code - so if you've previously installed this plugin, you'll need to reinstall it because its name has changed even at the plugin directory level.